networkx/algorithms/centrality/eigenvector.py

```diff
@@ -332,6 +342,10 @@ def eigenvector_centrality_numpy(G, weight=None, max_iter=50, tol=0):
         raise nx.NetworkXPointlessConcept(
             "cannot compute centrality for the null graph"
         )
+    connectivity_func = nx.is_strongly_connected if G.is_directed() else nx.is_connected
+    if not connectivity_func(G):  # See gh-6888.
```

Comment:
Just a naming suggestion, feel free to ignore!

```suggestion
    connected = nx.is_strongly_connected if G.is_directed() else nx.is_connected
    if not connected(G):  # See gh-6888.
```